types/src/front/lib/connectors_api.ts

```diff
@@ -528,16 +531,54 @@ export class ConnectorsAPI {
     };
   }
 
+  private async _fetchWithError(
```

Comment:
`nit`

No need for the `_` since it's a private method, it can't be used outside.

Reply:
Just following the convention that was there... not completely crazy so fine keeping it IMHO

Reply:
FWIW it was introduced a long long time ago when classes could not defined private properties/methods 🙃.